### ISSUE TITLE
[thepiratebay] Exception (thepiratebay): The message received was unexpected or badly formatted.: The SSL connection could not be established, see inner exception. (Test)

### ISSUE THREAD

**Issue by reader6886** [Jackett/Jackett#9281]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed.

Please read our [Contributing Guidelines](https://github.com/Jackett/Jackett/blob/master/CONTRIBUTING.md) before submitting your issue to ensure a prompt response to your bug.

### Environment

**OS**:

**.Net Runtime**: [.Net-Core/.Net-Framework/Mono]

**.Net Version**:

**Jackett Version**:

**Last Working Jackett Version**:

**Are you using a proxy or VPN?** [yes/no]

### Description

[List steps to reproduce the error and details on what happens and what you expected to happen]

### Logged Error Messages

[Place any relevant error messages you noticed from the logs here.]

[Make sure you attach the full logs with your personal information removed in case we need more information]

### Screenshots

[Place any screenshots of the issue here if needed]

**Comment by reader6886** [comment]
anyone knows any fix for this . piratebay throws this error when testing in jackett.

**Comment by reader6886** [comment]
tried with different vpns but the same error always.

**Comment by reader6886** [comment]
all other indexers are fine while testing one or two throw this eoor which is fine .But i haev never been able to successfully test piratebay in jackett with different urls or vpns. being able to add piratebay to jackett as indexer would be cool.
my specs are as follow

os -windows 10
jackett version - v0.16.909

this is the error detail in jackett log.

2020-08-03 11:51:51 | Error | {0} Jackett.Common.IndexerException: Exception (thepiratebay): The message received was unexpected or badly formatted.  ---> System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.  ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception.  ---> System.ComponentModel.Win32Exception (0x80090326): The message received was unexpected or badly formatted.    --- End of inner exception stack trace ---    at System.Net.Security.SslStream.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)

**Comment by garfield69** [comment]
looks very similar to #9264 
the issue is very likely a problem with the domains you are using being blocked by your ISP provider 
Assuming you are on at least Jackett  v0.16.838 - v0.16.940 (the newer the better),
then did you try accessing your jackett dashboard, and editing the thepiratebay indexer config and trying out each of the alternate sitelinks in turn to see if any of them work?
I should note that I tested all those alternate links and the work fine for me.
If you cannot get any of the ones in the config to work then you are out of luck.

**Comment by reader6886** [comment]
thank you so much garfield.
Jackett Version 0.16.909.0 .I am trying to udate to the 940.
this is what i did without any success.
1.tried using all the diff urls for piartebay on the indexer config setting.  same error
2. opening jackett in chrome and then using vpns from south korea and few other countries still jackett gave the same error while testing out the piratebay.
3. putting a working url(not yet blocked by isp) for piratbay on the jackett indexer config setting still the same error. the url opens up piratebay perfectly on any web browser . so i dont think it is to do with isp blocking as using vpn and other working urls give the same error.
3.Using the url for piratebay provided in jackett(tbp.party ) did work for 1 hour or so but after that gave the same error.

anyway i am happy with jackett i use it as standalone  torrent search tool too without connecting to radarr/sonarr . only if it could work the piratebay that would be icing on the cake.

if those links work on yours and dont on mine (even when using vpn ) is it something to do with ssl connection/certificate, how jackett treats the piratebay connection from my pc? not much idea about ssl and certificates etc.

thank you .

**Comment by garfield69** [comment]
>if those links work on yours and dont on mine (even when using vpn ) is it something to do with ssl connection/certificate, how jackett treats the piratebay connection from my pc?

Not likely an issue with your windows certificate store being out of date, as this is regularly updated.
But the problem is a clash between between client and host not agreeing on how the certificate specs should be, and this has nothing to do with Jackett.
Some external force is intefereing.

>Using the url for piratebay provided in jackett(tbp.party ) did work for 1 hour or so but after that gave the same error.

this suggest that the VPN session is being dropped or broken.
Have you tried using a different VPN service?

**Comment by reader6886** [comment]
thanks for the replies garfield.
it did not work with different vpn. i am not gonna bother too much about it now and hope one day it starts working magically...

if i can ask you about one other thing. so my jacket version is 
 Jackett Version 0.16.909.0 . 
when i try to update it says 
New release found. Current version: v0.16.909 New version: v0.16.962
Downloading release v0.16.962 It could take a while...

after some time i get usual notification saying jakcett has been updated to  v0.16.962 . i get excited restart the jackett and check the version it shows the same old Version 0.16.909.0. i have been through this atleast 6-7 times since trying to update to Version 0.16.940.0.
any fix or anything that i should do?

thank you so much .

**Comment by garfield69** [comment]
Hmm. I am not aware of any issues with windows updating, mine regularly updates without issues, I run on Win 10 home.
I would suggest shutting down the service and tray, and installing the latest manually from https://github.com/Jackett/Jackett/releases/download/v0.16.962/Jackett.Installer.Windows.exe
I recommend installing as a service, it removes possibility of clashes with active Jackett programs and updater file replacements.
 
Then keep an eye on it tomorrow when the next release gets published and see if the updater works.
If it fails then raise a new ticket and we can try to dig deeper.

**Comment by reader6886** [comment]
thanks so much i will try that .
